VKApp/VKApp/TabBar/Thrird/MainTableViewController.swift

```diff
@@ -62,6 +79,24 @@ final class MainTableViewController: UITableViewController {
             }
         }
     }
+
+    @objc private func refreshNewsfeedAction() {
+        var freshDate: TimeInterval?
+
+        guard let firstItem = postsItems.first?.date else { return }
+        freshDate = Double(firstItem) + 1
+        networkService
```

Comment:
```suggestion
        networkService
```
запрос в отдельный метод